python_gdeploy/wrapper/gdeploy_wrapper.py

```diff
@@ -0,0 +1,48 @@
+import os
+import subprocess
+import uuid
+
+GDEPLOY_CONFIG_PATH = "/var/run/python_gdeploy/config_"
```

Comment:
@TimothyAsir is working on building the spec file. I think he is taking care of this